### PR TITLE
eos: Add new states lldp_global and lldp_interfaces

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Related to #23 
Add gathered , parsed and rendered to eos_lldp_global and eos_lldp_interfaces

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: plugins/modules/eos_lldp_global.py [hunk 1]
```diff
@@ -96,6 +96,16 @@
             description:
             - Enable or disable system name TLV.
             type: bool
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
```
Comment (NilashishC) [inline plugins/modules/eos_lldp_global.py:102]:
```suggestion
      - The value of this option should be the output received from the EOS device by executing
```

File: tests/integration/targets/eos_lldp_global/tasks/main.yaml [hunk 2]
```diff
@@ -1,17 +1,7 @@
...
-
-- name: run test cases (connection=ansible.netcommon.network_cli)
-  include: '{{ test_case_to_run }} ansible_connection=ansible.netcommon.network_cli'
-  with_items: '{{ test_items }}'
-  loop_control:
-    loop_var: test_case_to_run
-  tags: connection_network_cli
+- include: cli.yaml
+  tags:
+    - cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_lldp_global/tasks/main.yaml:4]: network_cli

File: tests/integration/targets/eos_lldp_global/tasks/main.yaml [hunk 3]
```diff
@@ -1,17 +1,7 @@
...
+- include: eapi.yaml
+  tags:
+    - eapi
```
Comment (pabelanger) [inline tests/integration/targets/eos_lldp_global/tasks/main.yaml:7]: httpapi

File: tests/integration/targets/eos_lldp_interfaces/tasks/main.yaml [hunk 4]
```diff
@@ -1,17 +1,7 @@
...
-
-- name: run test cases (connection=ansible.netcommon.network_cli)
-  include: '{{ test_case_to_run }} ansible_connection=ansible.netcommon.network_cli'
-  with_items: '{{ test_items }}'
-  loop_control:
-    loop_var: test_case_to_run
-  tags: connection_network_cli
+- include: cli.yaml
+  tags:
+    - cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_lldp_interfaces/tasks/main.yaml:4]: network_cli

File: tests/integration/targets/eos_lldp_interfaces/tasks/main.yaml [hunk 5]
```diff
@@ -1,17 +1,7 @@
...
+- include: eapi.yaml
+  tags:
+    - eapi
```
Comment (pabelanger) [inline tests/integration/targets/eos_lldp_interfaces/tasks/main.yaml:7]: httpapi

**Pull request merged**